### PR TITLE
Fix scrollbar overlaying titlebar

### LINKED ISSUE #1835: [Bug]: Scrollbar appears in custom titlebar
### Preflight Checklist

- [X] I use the latest version of YouTube Music (Application).
- [X] I have searched the [issue tracker](https://github.com/th-ch/youtube-music/issues) for a bug report that matches the one I want to file, without success.
- [X] I understand that **th-ch/youtube-music has NO affiliation with Google or YouTube**

### YouTube Music (Application) Version

3.3.2

### Checklists

- [ ] I use the portable version of the YouTube Music Application.
- [ ] I can reproduce this issue in the [official version of (WEB) YTM](https://music.youtube.com).

### What operating system are you using?

Windows

### Operating System Version

Windows 11 version 23H2

### What CPU architecture are you using?

x64

### Last Known Working YouTube Music (Application) version

_No response_

### Reproduction steps

1. Enable the In-App-Menu (if its not already enabled)

### Expected Behavior

scrollbar stops at the bottom edge of the titlebar; no page content should escape the boundary

### Actual Behavior

image

### Enabled plugins

1. In-App-Menu
2. Album color theme
3. Blur nav
4. If any other plugins are relevant please let me know

### Additional Information

_No response_

### LINKED ISSUE #1982: The scrollbar appears beside the minimize, maximize and close buttons.
### Preflight Checklist

- [X] I use the latest version of YouTube Music (Application).
- [X] I have searched the [issue tracker](https://github.com/th-ch/youtube-music/issues) for a bug report that matches the one I want to file, without success.
- [X] I understand that **th-ch/youtube-music has NO affiliation with Google or YouTube**

### YouTube Music (Application) Version

3.3.6

### Checklists

- [ ] I use the portable version of the YouTube Music Application.
- [ ] I can reproduce this issue in the [official version of (WEB) YTM](https://music.youtube.com).

### What operating system are you using?

Windows

### Operating System Version

Windows 11 23H2

### What CPU architecture are you using?

x64

### Last Known Working YouTube Music (Application) version

_No response_

### Reproduction steps

Make sure in app menu is enabled.

### Expected Behavior

The  scroll bar should be below the buttons.
Screenshot (289)

### Actual Behavior

Screenshot (289)
As you can see in this photo the scroll bar is placed beside the buttons.

### Enabled plugins

1. ambient mode
2. in app menu
3. crossfade
4. taskbar media control
5. Discord Rich Presence

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by luluna02**
Fixes #1982 and fixes #1835

before: scrollbar overlaying titlebar

Screenshot 2024-04-20 at 3 59 28 PM

after fix:

https://github.com/th-ch/youtube-music/assets/133816965/4fcb0608-5b29-4fbb-b479-6f4799bf02b3

### CODE REVIEW

File: src/plugins/in-app-menu/renderer/TitleBar.tsx [hunk 1]
```diff
@@ -270,8 +270,21 @@ export const TitleBar = (props: TitleBarProps) => {
...
+    ytmusicAppLayout?.addEventListener("scroll",()=>{
+    const scrollValue = ytmusicAppLayout.scrollTop;
+    if (scrollValue > 20){
+      ytmusicAppLayout.classList.add("content-scrolled");
+    }
+    else{
+      ytmusicAppLayout.classList.remove("content-scrolled");
+    }
+    })
+
```
Comment (Su-Yong) [inline src/plugins/in-app-menu/renderer/TitleBar.tsx:283]: what is the function of `.content-scrolled`? I can't find `content-scrolled` class in `titlebar.css` file

Comment (luluna02) [reply src/plugins/in-app-menu/renderer/TitleBar.tsx:283]: This class is added/removed when you scroll the app. What I did is I made only the app layout scrollable and the whole app (only when the inappmenu is active) and this class serves to change the nav bars(top and side) background color. The class is already built by YouTube it's not new, that's why it's not present in the code.
Try the code and activate in app menu, scroll and watch the nav bars background color you'll get it.

**Comment by inesmrad** [comment]
@luluna02 I tried integrating your code and the scrollbar looks way less disturbing this way. Good work !

**Review by Su-Yong — APPROVED** [review]

**Comment by DereC4** [comment]
It's crazy how the css tag from back in May was causing all a huge performance issue 😭 (apparently)

**Pull request merged**